#include "scene_client_player_replied_handler.h"
#include "src/game_logic/thread_local/thread_local_storage.h"
#include "src/network/message_system.h"
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE
void ClientPlayerSceneServiceRepliedHandler::EnterSceneC2S(entt::entity player,
	const ::EnterSeceneC2SRequest* request,
	::EnterSeceneC2SResponse* response)
{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE
}

void ClientPlayerSceneServiceRepliedHandler::PushEnterSceneS2C(entt::entity player,
	const ::EnterSeceneS2C* request,
	::EnterSeceneS2C* response)
{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE
}

void ClientPlayerSceneServiceRepliedHandler::PushSceneInfoS2C(entt::entity player,
	const ::SceneInfoS2C* request,
	::SceneInfoS2C* response)
{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE
}

